tests/csi/README.md

```diff
@@ -0,0 +1,24 @@
+# CSI Binaries directory
```

Comment:
I believe that these should either leverage [git LFS](https://git-lfs.github.com/), or be hosted somewhere else. I would prefer not to download 70 MB when cloning the repository. (Maybe it already is and I just cant tell from the PR).

Reply:
I didn't do this as it looked like it was gonna be a hassle. Let me give it a try

Reply:
Wait, wouldn't this mean we'd have to set up LFS on every build run and anyone who wanted to run the tests locally would have to have it as well? I feel like I'd prefer storing the binary somewhere and downloading it on demand

Reply:
Hmmm, yes but this is pretty standard best practice, and easy to do. It’s pretty rude to have large binaries stored in git itself.

Reply:
Yes I agree. But I think that downloading is preferable to having git LFS for 2 files only used during tests. If these were key components, maybe. But I hesitate to add Yet Another Step™ to getting someone going with krustlet

Reply:
Oh, I see, downloading is fine. I thought you were proposing keeping it in git.

Reply:
FWIW, I'm pretty sure if its stored in LFS, you can still clone the repo, you would only need to set up LFS to obtain the binaries.

Reply:
Basically I am debating between having an on-demand GH action that builds these binaries and pushes them into the blobstore or just manually pushing them into the blob store. I'll probably revisit on Monday once I get some more random opinions from the internet. But I don't think git LFS is the right choice here. Removing them from git definitely is

Reply:
It looks like [csi-release-tools](https://github.com/kubernetes-csi/csi-release-tools) only provides Docker images.

Have you considered pulling the binary out of the container image? That would alleviate the need to compile from source.

```
><> id=$(docker create k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.2.0)
><> docker cp $id:/csi-node-driver-registrar - | tar xf - > ./csi-node-driver-registrar
><> chmod 755 csi-node-driver-registrar
><> ./csi-node-driver-registrar --help
Usage of ./csi-node-driver-registrar:
  -add_dir_header
        If true, adds the file directory to the header of the log messages
```

Reply:
That could work but adds a hard dependency on docker. I think pretty much everyone develops using `kind`, which means docker would be installed, but we haven't required that as of yet. I like keeping the set of tools small, but if this is easiest, we can do it this way

Reply:
If you're hard set on either github actions vs git-lfs, I would prefer setting up a GitHub Action that builds the binaries from source, then pushes them up as a Github Release. That way we can keep CSI artifacts and Krustlet artifacts separate. Then in Krustlet we'd download those artifacts during a CI run.

You could set up a separate repository for that... csi-release-action would seem like a good descriptive name.

Reply:
Ok, think I am going to keep it simple and have an on demand job that uploads them to the krustlet blobstore, purely for testing purposes. If people find having the compiled binaries around useful, we can break out a separate project

Reply:
https://github.com/deislabs/krustlet/pull/614

Reply:
Ok this is addressed